README.rst

```diff
@@ -1,6 +1,12 @@
 OpenSoar
```

Comment:
(not related to this line)
If all looks good, do final check with Pysoar on some comps